Penteract/Assets/Scripts/HUDManager.cpp

```diff
@@ -279,9 +284,21 @@ void HUDManager::StartCharacterSwitch() {
 					switchChildren[HIERARCHY_INDEX_SWITCH_ABILITY_IN_USE_GLOW]->Enable();
 				}
 			}
+
+			switchChildren[HIERARCHY_INDEX_SWITCH_ABILITY_GREEN_EFFECT]->Enable();
 		}
 	}
 	if (playingLostHealthFeedback) StopLostHealthFeedback();
+		
+	// Check if the new character needs the health warning
+
+	float health = fangObj->IsActive() ? onimaruPreviousHealth : fangPreviousHealth;
+	float maxHealth = fangObj->IsActive() ? playerController->GetOnimaruMaxHealth() : playerController->GetFangMaxHealth();
+
+	if (health > maxHealth * (criticalHealthPercentage / 100.f)) {
```

Comment:
What if we try to use 0.whatever instead of whatever/100.f

Reply:
Donezel Washington